components/cdc/src/endpoint.rs

```diff
@@ -819,10 +869,37 @@ impl<T: 'static + RaftStoreRouter<E>, E: KvEngine> Endpoint<T, E> {
         }
     }
 
+    // detect outlier raw regions, schedule deregister for outlier raw regions.
+    fn handle_raw_outlier_regions(
+        &self,
+        raw_resolved_regions: &ResolvedRegionHeap,
+        min_ts: TimeStamp,
+    ) {
+        let outliers = raw_resolved_regions.get_extreme_outlier(min_ts);
+        for region_id in outliers {
+            if let Some(delegate) = self.capture_regions.get(&region_id) {
+                let observe_id = delegate.handle.id;
+                let deregister = Deregister::Delegate {
+                    region_id,
+                    observe_id,
+                    err: Error::Other(box_err!("raw region dead lock")),
+                };
+                if let Err(e) = self.scheduler.schedule(Task::Deregister(deregister)) {
+                    error!("cdc schedule cdc task failed"; "error" => ?e);
+                }
+                CDC_RAW_REGION_OUTLIER_COUNT.inc();
```

Comment:
Suggest to also report the duration histogram of outlier raw resolved-ts.

Reply:
added.